shuffle/cli/src/transactions.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+use crate::shared::{get_home_path, Home};
+use anyhow::{anyhow, Result};
+use diem_types::account_address::AccountAddress;
+use serde_json::Value;
+use std::{cmp::max, fs, str::FromStr};
+use url::Url;
+
+use std::{thread, time};
+
+// Will list the last 10 transactions and has the ability to block and stream future transactions
+pub fn handle(network: Url, tailing: bool) -> Result<()> {
+    let home = Home::new(get_home_path().as_path())?;
+    let address_str = fs::read_to_string(home.get_latest_address_path())?;
+    let address = AccountAddress::from_str(address_str.as_str())?;
+
+    let path = network.join(format!("accounts/{}/transactions", address).as_str())?;
+    let seq_num = get_sequence_number(&network, address)?;
+    let start = max(seq_num as i64 - 10, 0);
+    let resp = ureq::get(path.as_str())
+        .query("start", start.to_string().as_str())
+        .query("limit", "10")
+        .call();
+    let json: serde_json::Value = resp.into_json()?;
+
+    let all_transactions = json
+        .as_array()
+        .ok_or_else(|| anyhow::anyhow!("Failed to get transactions"))?;
+
+    for txn_index in 0..all_transactions.len() {
+        println!("{:?}", json[txn_index]);
+        println!();
+        println!();
+    }
+
+    let mut old_seq_num = seq_num;
+
+    if tailing {
+        // listening for incoming transactions
+        loop {
+            old_seq_num = listen_for_next_txn(old_seq_num, &path, &network, address)?;
+        }
+    }
+    Ok(())
+}
+
+pub fn listen_for_next_txn(
+    old_seq_num: u64,
+    path: &Url,
+    network: &Url,
+    address: AccountAddress,
+) -> Result<u64> {
+    let seq_num = get_sequence_number(network, address)?;
+    thread::sleep(time::Duration::from_millis(1000));
+    // if transaction was sent the sequence number would automatically be updated to n + 1
+    // so we can get the nth sequence number's transaction
+    if seq_num > old_seq_num {
+        let resp = ureq::get(path.as_str())
+            .query("start", old_seq_num.to_string().as_str())
+            .query("limit", "1")
+            .call();
```

Comment:
As discussed offline, we have some concerns here we are hoping a refactor+tests will cover, including
1. if we have many txns, as opposed to just 1, what happens?
2. the new seq number might be more than a single page (ie:100) greater than old seq number, what happens? You could just do a limit of 100 and wait for the sleep above to retrigger and get the next batch.
3. Let's be sure to respond with the seq number thats the last txn we print, so we don't skip transactions on the next iteration